src/ducks/account/LoanProgress.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import LoanPercentageLine from 'ducks/account/LoanPercentageLine'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import { Caption } from 'cozy-ui/transpiled/react/Text'
+import { get } from 'lodash'
+import { Figure } from 'components/Figure'
+
+const getReimbursedAmount = account => {
+  const usedAmount = get(account, 'loan.usedAmount')
+  const remaining = -get(account, 'balance')
+
+  return usedAmount - remaining
+}
+
+const getReimbursedPercentage = account => {
+  const reimbursedAmount = getReimbursedAmount(account)
+  const totalAmount = get(account, 'loan.usedAmount')
+  const percentage = (reimbursedAmount / totalAmount) * 100
+
+  return percentage
+}
```

Comment:
I wonder if these utility methods should be move to accounts/helpers.js. This would make
them easier to use in services for example. Also it keeps the separation between models and view clear.